### PR TITLE
Bubble sort, quick sort and chaining(Hashing) in c

### PR DESCRIPTION
**Pull request by The-Hritik**

### CODE REVIEW

File: C/Chaining.c [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ return p;
+ }
+ p=p->next;
+ }
+ return NULL;
+ 
+}
+#endif /* Chains_h */
+#include <stdio.h>
+#include "Chains.h"
```
Comment (AkshayNachappa) [inline C/Chaining.c:54]: Please check upon this. Compilation error.

**Review by AkshayNachappa — CHANGES_REQUESTED** [review]

File: C/Quick Sort.c [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ j=partition(A,l,h);
+ QuickSort(A,l,j);
+ QuickSort(A,j+1,h);
+ }
+}
+int main()
+{
+ int A[]={11,13,7,12,16,9,24,5,10,3},n=10,i;
+ 
+ QuickSort(A,n);
```
Comment (AkshayNachappa) [inline C/Quick Sort.c:40]: Too few arguments

**Review by AkshayNachappa — CHANGES_REQUESTED** [review]

File: C/Quick Sort.c (hunk 2 above)
Comment (The-Hritik) [reply C/Quick Sort.c:40]: it is just for example. User may exceed it according to the need.

Comment (AkshayNachappa) [reply C/Quick Sort.c:40]: The function ```QuickSort ``` takes 3 arguments. 
Please provide 3 arguments or change the function to resolve compilation error.
Function call at Line 40 is passing only 2 arguments.

**Review by AkshayNachappa — APPROVED** [review]

**Comment by AkshayNachappa** [comment]
Will be deleting the error files and merging only bubble_sort.c

**Pull request merged**